definitions/divorce/json/CaseEvent/CaseEvent.json

```diff
@@ -2653,19 +2653,6 @@
     "PostConditionState": "AmendPetition",
     "SecurityClassification": "Public"
   },
-  {
-    "LiveFrom": "29/06/2020",
```

Comment:
you've moved it from core file to nonprod file.

But you haven't moved permissions for this event from core file to nonprod file.

It will fail on prod as we would try to grant permission to event that is undefined for prod now.

Reply:
It should be fine as the core file is used for both non-prod and prod :) 
It just means however that the authorisations will be the same on AAT and Prod which is what we need so it should be okay

Reply:
Possibly "RU" would be enough, but put "CRU" just in case

Reply:
If you'd want the solicitor to be able to trigger the even you'd need to add 'C' to the list. So i guess 'CRU' is right.